exercises/concept/role-playing-game/Scarb.toml

```diff
@@ -0,0 +1,7 @@
+[package]
+name = "role_playing_game"
+version = "0.1.0"
+edition = "2024_07"
```

Comment:
Is this the Cairo edition or the role playing game edition? Should that be 2025?